warehouse/models/mart/gtfs/_mart_gtfs_dims.yml

```diff
@@ -252,10 +294,14 @@ models:
       Definitions for the original GTFS fields are available at:
       https://gtfs.org/schedule/reference/#fare_productstxt.
     columns:
-      - name: key
-        description: |
-          Synthetic primary key constructed from `feed_key` and `fare_product_id`.
-        tests: *pk_tests_with_dups
+      - *schedule_dim_pk
+      - *schedule_dim_gtfs_key_with_dupe_flag
+      # some very large feeds have many duplicates, so row-level counts on this violate our test
```

Comment:
Should this comment be moved up to where the anchor is defined? Specifically to define why you're excluding the MTC regional feed? And also, why exclude that feed from *all* such tests? Seems a bit odd?

Reply:
I will clarify in the YAML; I found this was causing multiple natural key tests to fail the 50% check since it's a very large feed that can dominate record-level tests that check for count or proportion

Reply:
Oh wait sorry this specific comment is specific to `dim_fare_products`; I will still clarify in both places